drivers/LVHDoHBASR.py

```diff
@@ -51,31 +53,29 @@
     'driver_version': '1.0',
     'required_api_version': '1.0',
     'capabilities': CAPABILITIES,
-    'configuration': CONFIGURATION
+    'configuration': CONFIGURATION,
     }
 
 
 class LVHDoHBASR(LVHDSR.LVHDSR):
     """LVHD over HBA storage repository"""
 
-    def handles(type):
+    def handles(type_):
         if __name__ == '__main__':
             name = sys.argv[0]
         else:
             name = __name__
-        if name.endswith("LVMoHBASR"):
-            return type == "lvmohba"  # for the initial switch from LVM
-        if type == "lvhdohba":
-            return True
-        return False
+
+        # for the initial switch from LVM
+        return type_ == "lvmohba" if name.endswith("LVMoHBASR") else "lvhdohba"
```

Comment:
This needs some parenthesis, as it is currently written it'll return either a boolean or a string.
Some 'mypy' type annotations might be useful in catching this kind of bug.